src/components/mortgage-investment-compare/amortization-table.js

```diff
@@ -0,0 +1,156 @@
+import React from 'react'
+
+import { Table } from 'reactstrap'
+import { LIGHT_GRAY, GRAY, LIGHT_GREEN } from '../../assets/colors'
+import { css, cx } from 'react-emotion'
+
+import { formatMoney } from '../../utils/numberUtils'
+
+const amortizationTable = css`
+  thead tr:first-of-type th {
+    font-size: 1rem;
+  }
+
+  th,
+  tbody {
+    text-align: center;
+    font-size: 0.75rem;
+  }
+
+  th:first-of-type,
+  td:first-of-type {
+    white-space: nowrap;
+  }
+
+  th:first-of-type {
+    border: 0;
+  }
+
+  th:nth-child(6),
+  td:nth-child(6) {
+    border-left-width: thick;
+    border-left-color: ${GRAY}};
+  }
+`
+
+const checkLoanTerms = (year, option1, option2) => {
+  return year + 1 === option1.term || year + 1 === option2.term
+}
```

Comment:
I'm a bit lost here 2nd time around. What does `checkLoanTerms` do? Perhaps some documentation could help?

Reply:
it checks if the current row is the 15th or 30th year row, before highlighting. Used only in `highlightTableCells`. Maybe I can move it inside the function and add some documentation?